src/sv-pipeline/04_variant_resolution/scripts/clean_vcf_part5_update_records.py

```diff
@@ -123,18 +123,22 @@ def main():
 
                 if gt5kb_del:
                     for sample_obj in record.samples.itervalues():
-                        if not sample_obj['GQ'] is None and sample_obj['RD_CN'] >= 2:
+                        if not sample_obj['GQ'] is None and \
+                                (sample_obj['RD_CN'] is not None and sample_obj['RD_CN'] >= 2):
                             sample_obj['GT'] = (0, 0)
-                        elif not sample_obj['GQ'] is None and sample_obj['RD_CN'] == 1:
+                        elif not sample_obj['GQ'] is None and \
+                                (sample_obj['RD_CN'] is not None and sample_obj['RD_CN'] == 1):
```

Comment:
Would the following syntax be a good alternative?

```python
int(sample_obj['RD_CN'] or 0) == 1
```

Reply:
Oh that's nifty!